src/main/java/net/rcarz/jiraclient/tempo/TeamMembership.java

```diff
@@ -104,19 +179,35 @@ public void setStatus(String status) {
         this.status = status;
     }
 
-    public Date getStartDate() {
+    public String getStartDate() {
         return startDate;
     }
 
-    public void setStartDate(Date startDate) {
+    public void setStartDate(String startDate) {
         this.startDate = startDate;
     }
 
-    public Date getEndDate() {
+    public String getEndDate() {
```

Comment:
Such changes are actually dangerous, since it's breaking of backward compatibility: for example my utility that grabs all teams with members from jira will stop working now :)
But I'm ok to upgrade just to move faster